src/components/app/BottomBox.tsx

```diff
@@ -150,6 +154,25 @@ class BottomBoxImpl extends React.PureComponent<Props> {
   _sourceView = React.createRef<SourceView>();
   _assemblyView = React.createRef<AssemblyView>();
 
+  constructor(props: Props) {
+    super(props);
+    this._onKeyDown = this._onKeyDown.bind(this);
```

Comment:
You won't have to bind this if you convert the _onKeyDown function into an arrow function. Like:
```ts
  _onKeyDown = (event: KeyboardEvent) => {
     if (event.key === 'Escape' && this.props.isFullscreen) {
      this.props.toggleBottomBoxFullscreen();
    }
  };

```